jwcrypto/jwk.py

```diff
@@ -332,6 +332,9 @@ def _get_curve_by_name(self, name):
             return ec.SECP384R1()
         elif name == 'P-521':
             return ec.SECP521R1()
+        elif name in ['Ed25519', 'Ed448', 'X25519', 'X448']:
```

Comment:
All you need to do here is return the same name you got in input as we need no mapping of names in these cases.

Reply:
Another option is to rename this to _get_ec_curve_by_name() and then simply not handle edwards curves in here.
After all this is an internal function and you are not using it in _generate_OKP() anyway.

Reply:
I'll rename it.

Reply:
Actually, it's useful for unit testing OKP keys so I'll follow your first suggestion.